autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -294,9 +292,9 @@ def train_and_save(self, X_train, y_train, X_test, y_test, model: AbstractModel,
                 self.model_types_inner[model.name] = model._child_type
             else:
                 self.model_types_inner[model.name] = type(model)
-            logger.log(20, '\t' + str(round(fit_end_time - fit_start_time, 2))+'s' + '\t = Training runtime')
             logger.log(20, '\t' + str(round(score, 4)) + '\t = Validation ' + self.objective_func.name + ' score')
-            logger.log(15, '\tEvaluation runtime of '+str(model.name)+ ' = '+str(round(pred_end_time - fit_end_time, 2))+' s')
+            logger.log(20, '\t' + str(round(fit_end_time - fit_start_time, 2)) + 's' + '\t = Training runtime')
+            logger.log(20, '\t' + str(round(pred_end_time - fit_end_time, 2))+'s' + '\t = Validation runtime')
```

Comment:
consider more informative message than "Validation runtime", maybe:
"Validation runtime (making predictions and evaluating them)"

Reply:
For now lets keep it simple, we can revisit the phrasing later on in a dedicated logging improvement PR.